daemon/daemon.go

```diff
@@ -1207,7 +1207,7 @@ func (daemon *Daemon) ImageGetCached(imgID string, config *runconfig.Config) (*i
 
 func checkKernelAndArch() error {
 	// Check for unsupported architectures
-	if runtime.GOARCH != "amd64" {
+	if runtime.GOARCH != "amd64" && runtime.GOARCH != "ppc64" && runtime.GOARCH != "ppc64le" && runtime.GOARCH != "s390x" {
 		return fmt.Errorf("The Docker runtime currently only supports amd64 (not %s). This will change in the future. Aborting.", runtime.GOARCH)
```

Comment:
Shouldn't the message be changed as well?  But, not sure it needs to list everything that is (going to be) supported; maybe "The Docker runtime is not enabled for architecture %q."